sale-matome/src/store.tsx

```diff
@@ -0,0 +1,12 @@
+import { createStore, applyMiddleware } from 'redux'
+import { composeWithDevTools } from 'redux-devtools-extension'
+import thunk from 'redux-thunk'
+import rootReducer from './reducers'
+
+const store = createStore(rootReducer, composeWithDevTools(applyMiddleware(thunk)))
+
+console.log(store.getState())
+
+export type RootState = ReturnType<typeof store.getState>
```

Comment:
```suggestion
const store = createStore(rootReducer, composeWithDevTools(applyMiddleware(thunk)))

export type RootState = ReturnType<typeof store.getState>
```

不要なconsole.logを削除しましょう。